manifests/vhost.pp

```diff
@@ -362,7 +362,7 @@
   }
 
   # Load mod_alias if needed and not yet loaded
-  if ($scriptalias or $scriptaliases != []) or ($redirect_source and $redirect_dest) {
+  if ($scriptalias or $scriptaliases != []) or ($aliases or $aliases != []) or ($redirect_source and $redirect_dest) {
```

Comment:
Changing to `and` makes more sense, but the initial patch was imitating what was already in place (or). I'll update, and push shortly.